site2/docs/schema-understand.md

```diff
@@ -287,25 +289,36 @@ This example shows how to construct a key/value schema and then use it to produc
     KeyValue<Integer, String> kv = msg.getValue();
     ```
 
+<!--END_DOCUSAURUS_CODE_TABS-->
+
 #### struct
 
-Pulsar uses [Avro Specification](http://avro.apache.org/docs/current/spec.html) to declare the schema definition for `struct` schema. 
+This section describes the details of type and usage of the `struct` schema.
 
-This allows Pulsar:
+##### Type
 
-* to use same tools to manage schema definitions
+`struct` schema supports `AvroBaseStructSchema` and `ProtobufNativeSchema`.
 
-* to use different serialization/deserialization methods to handle data
+|Type|Description|
+---|---|
+`AvroBaseStructSchema`|Pulsar uses [Avro Specification](http://avro.apache.org/docs/current/spec.html) to declare the schema definition for `AvroBaseStructSchema`, which supports  `AvroSchema`, `JsonSchema`, and `ProtobufSchema`. <br><br>This allows Pulsar:<br>- to use same tools to manage schema definitions<br>- to use different serialization or deserialization methods to handle data|
+`ProtobufNativeSchema`|`ProtobufNativeSchema` is based on protobuf native Descriptor. <br><br>This allows Pulsar:<br>- to use native protobuf-v3 to serialize or deserialize data<br>- to use `AutoConsume` to deserialize data.
 
-There are two methods to use `struct` schema：
+##### Usage
+
+Pulsar provides three methods to use the `struct` schema:
```

Comment:
Technically, it's 100% correct.
I suggest that we use "Pulsar provides the following methods to ...."
When people add more methods below, they usually forget to update the specific number.